docs/api.md

```diff
@@ -51,7 +48,40 @@ All the original static methods of the component are hoisted.
 
 ###### `getWrappedInstance(): ReactComponent`
 
-Returns the wrapped component instance. Only available if you pass `{ withRef: true }` as part of the `connect()`’s second `options` argument.
+Returns the wrapped component instance. Only available if you set `connect.defaults({ withRef: true })`.
+
+### `connect.defaults([newDefaults])`
+
+Returns a new `connect` which will have its defaults amended according to `newDefaults`.
+
+Calls to `.defaults()` can be chained, each latter call overriding defaults set in previous ones:
+
+```js
+const withRef = connect.defaults({ withRef: true })
+const withRefAndWithPost = withRef.defaults({ method: 'POST' })
+const onlyWithPost = withRefAndWithPost.defaults({ withRef: false })
+```
+
+#### Arguments
+
+* `[newDefaults = {}]` *(Object)*: An object with any of the following keys:
+     - `buildRequest(mapping): Request` *(Function)*: Takes a mapping and returns a `Request`. If setting this, make sure it interoperates with either the default `fetch` and `Request` or the `fetch` and `Request` you've provided. Defaults to the internal implementation, see the source for details.
```

Comment:
will this be after coercion? in other words, will this function need to handle raw `url` strings or will `mapping` always be an object? I'm guess the latter, but I'm purposely reading the docs before looking at the source to have an outsider's mind.